slrealizer/tests/test_om10realizer.py

```diff
@@ -31,13 +29,17 @@ class OM10RealizerTest(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        from slrealizer import data
```

Comment:
This is nice! But could the `data_dir` variable be set in this module as well? Or maybe some standard items like `data.twinkles_observation_catalog` etc?

Reply:
After reading into some ways of doing this, I've implemented a `Dataloader` class in our `data` subpackage following `scipy.misc` [way](https://github.com/scipy/scipy/tree/master/scipy/misc) of interfacing with package data. What do you think? I like that it
- modularizes all the preprocessing of data such as querying
- provides a place where we can document different ways we preprocess data (such as painting the lenses and quasars in OM10)
- declutters the catalog-generating scripts